cirq/linalg/decompositions.py

```diff
@@ -514,6 +528,120 @@ def interaction_matrix(m: np.ndarray, c: float) -> np.ndarray:
             before)
 
 
+def scatter_plot_normalized_kak_interaction_coefficients(
+        interactions: Iterable[
+            Union[np.ndarray, 'cirq.SupportsUnitary', 'KakDecomposition']],
+        *,
+        include_frame: bool = True,
+        ax: Optional[plt.Axes] = None,
+        show: bool = False,
+        **kwargs):
+    """Plots the interaction coefficients of many two-qubit operations.
+
+    Plots:
+        A point for the (x, y, z) normalized interaction coefficients of
+        each interaction from the given interactions. The (x, y, z) coordinates
+        are normalized so that the maximum value is at 1 instead of at pi/4.
+
+        If `include_frame` is set to True, then a black wireframe outline of the
+        canonicalized normalized KAK coefficient space. The space is defined by
+        the following two constraints:
+
+            0 <= abs(z) <= y <= x <= 1
+            if x = 1 then z >= 0
+
+        The wireframe includes lines along the surface of the space at z=0.
+
+    Args:
+        interactions: An iterable of two qubit unitary interactions. Each
+            interaction can be specified as a raw 4x4 unitary matrix, or an
+            object with a 4x4 unitary matrix according to `cirq.unitary` (
+            (e.g. `cirq.CZ` or a `cirq.KakDecomposition` or a `cirq.Circuit`
+            over two qubits).
+        include_frame: Determines whether or not to draw the kak space
+            wireframe. If not set. Defaults to `True`.
```

Comment:
nit: "If not set, defaults to `True`." (one sentence, two sub-clauses).

Alternatively, remove this since it duplicates what is already stated in function signature.